x/launch/keeper/msg_server_settle_request_test.go

```diff
@@ -49,103 +57,143 @@ func TestMsgSettleRequest(t *testing.T) {
 		{
 			name: "invalid chain",
 			msg: types.MsgSettleRequest{
-				LaunchID:    invalidChain,
-				Coordinator: coordinator1.Address,
-				RequestID:   requests[0].RequestID,
-				Approve:     true,
+				LaunchID:  invalidChain,
+				Settler:   coordinator1.Address,
+				RequestID: requests[0].RequestID,
+				Approve:   true,
 			},
 			err: types.ErrChainNotFound,
 		},
 		{
 			name: "launch triggered chain",
 			msg: types.MsgSettleRequest{
-				LaunchID:    chains[0].LaunchID,
-				Coordinator: coordinator1.Address,
-				RequestID:   requests[0].RequestID,
-				Approve:     true,
+				LaunchID:  chains[0].LaunchID,
+				Settler:   coordinator1.Address,
+				RequestID: requests[0].RequestID,
+				Approve:   true,
 			},
 			err: types.ErrTriggeredLaunch,
 		},
 		{
 			name: "coordinator not found",
 			msg: types.MsgSettleRequest{
-				LaunchID:    chains[1].LaunchID,
-				Coordinator: coordinator1.Address,
-				RequestID:   requests[0].RequestID,
-				Approve:     true,
+				LaunchID:  chains[1].LaunchID,
+				Settler:   coordinator1.Address,
+				RequestID: requests[0].RequestID,
+				Approve:   true,
 			},
 			err: types.ErrChainInactive,
 		},
 		{
 			name: "no permission error",
 			msg: types.MsgSettleRequest{
-				LaunchID:    chains[2].LaunchID,
-				Coordinator: coordinator2.Address,
-				RequestID:   requests[0].RequestID,
-				Approve:     true,
+				LaunchID:  chains[2].LaunchID,
+				Settler:   coordinator2.Address,
+				RequestID: requests[0].RequestID,
+				Approve:   true,
 			},
 			err: types.ErrNoAddressPermission,
 		},
 		{
 			name: "approve an invalid request",
 			msg: types.MsgSettleRequest{
-				LaunchID:    chains[2].LaunchID,
-				Coordinator: coordinator1.Address,
-				RequestID:   99999999,
-				Approve:     true,
+				LaunchID:  chains[2].LaunchID,
+				Settler:   coordinator1.Address,
+				RequestID: 99999999,
+				Approve:   true,
 			},
 			err: types.ErrRequestNotFound,
 		},
 		{
-			name: "approve chain request 1",
+			name: "coordinator approves chain request 1",
 			msg: types.MsgSettleRequest{
-				LaunchID:    chains[2].LaunchID,
-				Coordinator: coordinator1.Address,
-				RequestID:   requests[0].RequestID,
-				Approve:     true,
+				LaunchID:  chains[2].LaunchID,
+				Settler:   coordinator1.Address,
+				RequestID: requests[0].RequestID,
+				Approve:   true,
 			},
 			checkAddr: addr1,
 		},
 		{
-			name: "approve chain request 2",
+			name: "coordinator approves chain request 2",
 			msg: types.MsgSettleRequest{
-				LaunchID:    chains[2].LaunchID,
-				Coordinator: coordinator1.Address,
-				RequestID:   requests[1].RequestID,
-				Approve:     true,
+				LaunchID:  chains[2].LaunchID,
+				Settler:   coordinator1.Address,
+				RequestID: requests[1].RequestID,
+				Approve:   true,
 			},
 			checkAddr: addr2,
 		},
 		{
-			name: "approve chain request 3",
+			name: "coordinator approves chain request 3",
```

Comment:
sorry it's not related directly to the scope of the solved issue but it seems `request 3` and `request 4` are unnecessary since with `request 2` we already test approving a second request on a chain
It seems to be a problem now that we add new tests and make this test file harder to maintain
I would consider removing them